src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerIT.java

```diff
@@ -81,15 +81,17 @@ public static void ensurePaAndRcaEnabled() throws Exception {
             Response resp = client().performRequest(new Request("GET", "_opendistro/_performanceanalyzer/cluster/config"));
             Map<String, Object> respMap = mapper.readValue(EntityUtils.toString(resp.getEntity(), "UTF-8"),
                     new TypeReference<Map<String, Object>>(){});
-            if (respMap.get("currentPerformanceAnalyzerClusterState").equals(3)) {
+            if (respMap.get("currentPerformanceAnalyzerClusterState").equals(3) &&
```

Comment:
Shouldn't it be this?
```
respMap.get("currentPerformanceAnalyzerClusterState").equals(3) ||
                    respMap.get("currentPerformanceAnalyzerClusterState").equals(7)
```

Reply:
That's inverted logic, I'm detecting when PA and RCA aren't enabled so I can throw an exception